<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WifiConnectionWidget</class>
 <widget class="QWidget" name="WifiConnectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="SSIDLabel">
     <property name="text">
      <string>&amp;SSID</string>
     </property>
     <property name="buddy">
      <cstring>SSIDLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="SSIDLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="modeLabel">
     <property name="text">
      <string>&amp;Mode:</string>
     </property>
     <property name="buddy">
      <cstring>modeComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="modeComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Infrastructure</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ad-hoc</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Access Point</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Device MAC address:</string>
     </property>
     <property name="buddy">
      <cstring>macAddress</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="macAddress">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="inputMask">
      <string notr="true">HH:HH:HH:HH:HH:HH;_</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;Cloned MAC address:</string>
     </property>
     <property name="buddy">
      <cstring>clonedMacAddress</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="clonedMacAddress">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="inputMask">
      <string notr="true">HH:HH:HH:HH:HH:HH;_</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>M&amp;TU:</string>
     </property>
     <property name="buddy">
      <cstring>mtu</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="BSSIDLabel">
     <property name="text">
      <string>&amp;BSSID:</string>
     </property>
     <property name="buddy">
      <cstring>BSSIDLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="BSSIDLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="inputMask">
      <string notr="true">HH:HH:HH:HH:HH:HH;_</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="mtu">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="specialValueText">
      <string>Automatic</string>
     </property>
     <property name="suffix">
      <string> bytes</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>SSIDLineEdit</tabstop>
  <tabstop>modeComboBox</tabstop>
  <tabstop>BSSIDLineEdit</tabstop>
  <tabstop>macAddress</tabstop>
  <tabstop>clonedMacAddress</tabstop>
  <tabstop>mtu</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
